test/libsolidity/syntaxTests/nameAndTypeResolution/shift_warn_literal_large_shift_amount.sol

```diff
@@ -0,0 +1,9 @@
+contract test {
+    function f() pure public returns(uint) {
+        uint x = 100;
+        return 10 << x;
+    }
+}
+// ----
+// Warning: (98-105): Result of shift has type uint8 and thus might overflow. Silence this warning by converting the literal to the expected type.
+// Warning: (98-105): The result type of the shift operation is equal to the type of the first operand (uint8) ignoring the (larger) type of the second operand (uint256) which might be unexpected. Silence this warning by either converting the first or the second operand to the type of the other.
```

Comment:
Turns out there are even two different warnings :)

Reply:
Is it fine to have both warnings here? Maybe it is ok to have just second one if possible :-)

Reply:
One is about literals, the other about integer types, so I think it is fine.